docs/spec.md

```diff
@@ -2409,19 +2412,14 @@ Encapsulates an implementation-defined operation `call_target_name` that takes
 `backend_config` and `api_version` may be used to provide additional
 implementation-defined metadata.
 
-At the moment, this operation contains a fairly disorganized collection of
-metadata which reflects organic evolution of its counterpart operation in
-the XLA compiler. In the future, we are planning to unify this metadata
-([#741](https://github.com/openxla/stablehlo/issues/741)).
-
 #### Inputs
 
 | Label | Name                  | Type                                          |
 |-------|-----------------------|-----------------------------------------------|
 | (I1)  | `inputs`              | variadic number of values                     |
 | (I2)  | `call_target_name`    | constant of type `string`                     |
 | (I3)  | `has_side_effect`     | constant of type `i1`                         |
-| (I4)  | `backend_config`      | constant of type `string`                     |
+| (I4)  | `backend_config`      | constant of type `string` or attribute dictionary                   |
```

Comment:
We do not have a type in the spec (https://github.com/openxla/stablehlo/blob/main/docs/spec.md#types) to express dictionary. 

replace
```
or attribute dictionary
```
with
```
dictionary with key of type  `string` and values of type  `Value`
```